addons/fleet/views/fleet_vehicle_incident_views.xml

```diff
@@ -0,0 +1,110 @@
+            </search>
+        </field>
+    </record>
+
+    <record id="fleet_vehicle_incident_view_kanban" model="ir.ui.view">
+        <field name="name">fleet.vehicle.incident.kanban</field>
+        <field name="model">fleet.vehicle.incident</field>
+        <field name="arch" type="xml">
+            <kanban default_group_by="state">
+                <templates>
+                    <t t-name="card">
+                        <div>
+                            <div>
+                                <strong><field name="vehicle_id"/></strong>
+                            </div>
+                            <div>
+                                <field name="date"/>
+                            </div>
+                            <div class="d-flex gap-1">
+                                <field name="driver_id" widget="many2one_avatar"/>
+                                <field name="driver_id" class="small pt-1 pb-1"/>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id='fleet_vehicle_incident_view_list' model='ir.ui.view'>
+        <field name="name">fleet.vehicle.incident.list</field>
+        <field name="model">fleet.vehicle.incident</field>
+        <field name="arch" type="xml">
+            <list string="Incident" multi_edit="1">
+                <field name="date"/>
+                <field name="vehicle_id"/>
+                <field name="driver_id" widget="many2one_avatar"/>
+                <field name="incident_type_id"/>
+            </list>
+        </field>
+    </record>
+
+    <record id='fleet_vehicle_incident_view_form' model='ir.ui.view'>
+        <field name="name">fleet.vehicle.incident.form</field>
+        <field name="model">fleet.vehicle.incident</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="state"  widget="statusbar" options="{'clickable': '1'}"/>
+                </header>
+                <sheet>
+                    <group>
+                        <group string="Incident Details">
+                            <field name="vehicle_id"/>
+                            <field name="date"/>
+                            <field name="incident_type_id"/>
+                            <field name="driver_id" widget="many2one_avatar"/>
+                            <field name="is_third_party_involved"/>
+                            <field name="location" placeholder="Provide an accurate description of the location of the incident"/>
+                            <field name="description" placeholder="Provide an accurate description of the incident"/>
+                        </group>
+                        <group string="Third-party details" invisible="not is_third_party_involved">
```

Comment:
Should we add description and third-party in the tag? You can discuss with the PO

company is missing